app/views/work_items/index.html.erb

```diff
@@ -13,7 +13,13 @@
 
     <%= govuk_details(summary_text: t('.check_totals')) do %>
       <%=
-        head = [t('.items'), t('.total_cost'), t('.total_time')]
+        head = [
+          t('.items'),
+          t('.requested_time'),
+          t(".requested_cost.#{travel_and_waiting.vat_registered? ? 'inc' : 'exc'}_vat_html"),
```

Comment:
**nit** no biggee as may repos are guilty of it but i always have concerns on interpolation for translation lookup as it renders testing for missing translations, like with the [i18n-tasks gem](https://rubygems.org/gems/i18n-tasks/versions/1.0.12?locale=en), not possible.

I would not do now, but could extract to helper.

Reply:
not a bad shout as just realised I missed a translation in a previous PR.. :( 

though I am interested in see how you would extract this to a helper. :)